Feature/sponsor section

deweydell opened the pull request.

deweydell committed:
Update brand colors in custom.css

deweydell committed:
Remove body classes for static page

- Add page-intro class on countdown section
- Allow page to be scrollable

deweydell committed:
Add sponsor section layout

- Add flex box grid cdn
- Add breakpoints for grid for reference in custom.css
- Add placeholders for sponsor logos

deweydell committed:
Add current sponsors, remove template placeholders

deweydell committed:
Add link colors to sponsor sections

deweydell committed:
Fix typo

deweydell committed:
Remove two unused classes

jarmstrng commented on index.html:
```diff
@@ -12,94 +12,124 @@
+	    </div>
+	    <div class="col-xs-6 col-sm-3">
+	      <img src="assets/img/logos/puppet-logo.png" />
+	    </div>
```
can we get these logos as links to the company websites? the urls are in the story

also, these companies are contributing somewhere between 1/3-1/5 of what the bronze supporters are, so is it possible to make their logos smaller? if you try it and it looks absurd on mobile, just tell me to take a hike ;)

deweydell replied:
Yes I can make them smaller :) 👍  I think you're right it doesn't make sense to have them less than 2 per row on mobile for visibility, but for tablet/desktop they can be physically smaller!

jarmstrng commented on index.html:
```diff
@@ -12,94 +12,124 @@
+	    <div class="col-xs-12">
+	      <h3>Silver Level Sponsors</h3>
+	    </div>
+	    <div class="col-xs-12 col-sm-4">
```
is it possible/easy with the flex grid to center the sponsor boxes? it feels kinda empty on the right side of the page...

deweydell replied:
There is an easy way to center! You add a class to the row container of "center-xs" or -sm or -md or -lg depending on what breakpoint you want the row to be centered.

However, aesthetically I'm not sure it looks better centered because there is still awkward space around the sides. 
![screen shot 2017-02-20 at 10 00 22 am](https://cloud.githubusercontent.com/assets/11186572/23130228/9bcb1ed8-f753-11e6-8614-6c43546492bc.png)

I'm wondering if a better solution would be to add placeholder text/images/boxes that say "Your logo here! etc" and maybe subtly link to the prospectus? What do you think?

jarmstrng commented on index.html:
```diff
@@ -12,94 +12,124 @@
+	    </div>
+	    <div class="col-xs-12 col-md-6 end-md">
+	      <a href="sponsor-prospectus.pdf" target="_blank" class="prospectus-link">Interested in becoming a sponsor?</a>
+	    </div>
```
i just noticed that the button in our top section is very rounded, and this button is less rounded. the template isn't consistent but i think we should be. would you mind picking a button-roundedness and changing the existing buttons to that? i'll defer to your judgement on how rounded we should be ;)

jarmstrng replied:
(love the hover color, btw :D )

deweydell replied:
:) I'm a fan of less rounded! I noticed the buttons in the template are slightly rounded and just that "buy tickets" button in the countdown section is very rounded. So I can definitely alter that one.

jarmstrng commented on assets/css/custom.css:
```diff
@@ -69,21 +80,82 @@ body, #qcSiteNav h4, #qcPriNav ul > li > a, #qcPriNav ul ul > li > a, .dash_titl
+
+.sponsors-section .prospectus-link:hover {
+	background-color: #EB6851;
+}
```
maybe we can change the button classes to be reusable, like `.navy-button:hover` or `.dark-button:hover` or whatnot. then we'll be able to add the same class to to other buttons 

(i wish we'd asked for "dark button on light background" and "light button on dark background" designs from our designer... next year!)

jarmstrng replied:
(same goes for other potentially reusable classes)

deweydell replied:
Ok cool will do, I was going to ask about making some global classes, but I wasn't sure how people would want to handle design elements, so I figured I'd make everything specific to the sponsor section at first.

jarmstrng commented on assets/css/custom.css:
```diff
@@ -69,21 +80,82 @@ body, #qcSiteNav h4, #qcPriNav ul > li > a, #qcPriNav ul ul > li > a, .dash_titl
 
-body.home.static {
+.page-intro {
 	background: url('../img/WSC-bg.jpg') no-repeat center center fixed;
```
thank you for this name, my calling it the "splash page" for lack of better term was just awkward 😂

deweydell replied:
hahah!

jarmstrng reviewed (changes requested):
love this design!!! thanks so much for tackling this

jarmstrng commented:
Ooh, can we also move the prospectus to the /assets folder?

deweydell commented:
@jarmstrng we could move prospectus to the assets folder, but then it would be at 2017.writespeakcode.com/assets/prospectus.pdf

assuming that's ok with the rest of the team and they haven't been linking people to 2017.writespeakcode.com/prosepctus.pdf right?

deweydell committed:
Add review requests to sponsors section

- Replace cdk logo
- Add button and link global classes
- Add placeholder quote section to fill up space until we get more logos
- Move prospectus pdf to assets folder

jarmstrng commented:
@deweydell Asked about moving the prospectus in #2017_sponsorship - I think it'll be ok but you're right that we should double check

jarmstrng commented on index.html:
```diff
@@ -106,27 +106,32 @@ <h1 id="slidecaption">Write/Speak/Code 2017 conference</h1>
+				<p>Sponsorship is the most effective way to recruit talented, driven women to fill key technical positions and demonstrate your commitment to the professional development of women at your organization as well as the industry at large.</p>
+				<p>At Write/Speak/Code you can narrowly target women software engineers who are motivated to take their careers, and your company, to the top of the industry. Our attendees range from undergraduate students to CTOs.</p>
+				 <a href="assets/sponsor-prospectus.pdf">Read our 2017 Prospectus</a>
+			</div>
```
brilliant!

jarmstrng commented:
@deweydell :shipit:

deweydell committed:
Add spacing between logo and sponsor quote
